public/index.html

```diff
@@ -48,9 +48,8 @@
         Launch app
       </a>
       <footer class="mb-5 mt-auto text-gray-600 text-sm font-medium">
-        Copyright © 2022-<span id="currentYear">2024</span> VUSD. All Rights Reserved. Sponsored by <a
-          href="https://vesper.finance" rel="noreferrer" target="_blank"><span
-            class="text-indigo-500">Vesper</span></a>.
+        Copyright © 2021-<span id="currentYear">2024</span> VUSD. All Rights Reserved. Sponsored by <a
```

Comment:
current year is 2025? 😅 
```suggestion
        Copyright © 2021-<span id="currentYear">2025</span> VUSD. All Rights Reserved. Sponsored by <a
```

Reply:
A few lines below, a small script will update `currentYear` so updating the hardcoded value it is not strictly needed.

Reply:
In my defense, it was not visible 😄 

<img width="1597" alt="image" src="https://github.com/user-attachments/assets/d7efcc0d-b8ff-475e-90e5-3d7ae138febf" />